js/hitbtc3.js

```diff
@@ -1321,6 +1322,12 @@ module.exports = class hitbtc3 extends Exchange {
     async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
         await this.loadMarkets ();
         const market = this.market (symbol);
+        const reduceOnly = this.safeValue (params, 'reduce_only');
```

Comment:
Let's support the exchange specific name `reduce_only` as well as the unified parameter name `reduceOnly` with safeValue2.